src/Quadrature.h

```diff
@@ -0,0 +1,217 @@
+/**
+ * @file Quadrature.h
+ * @brief The Quadrature abstract class and subclasses.
+ * @details The Quadrature subclasses are defined with tabulated or functional
+ *          quadrature sets given in the "Lattice Physics Computations",
+ *          Handbook of Nuclear Engineering, Dave Knott, Akio Yamamoto, 2010.
+ * @date April 6, 2015
```

Comment:
You can update this date and put your name/email here if you wish (or remove it altogether)